vm/gvisor/gvisor.go

```diff
@@ -60,6 +61,11 @@ func ctor(env *vmimpl.Env) (vmimpl.Pool, error) {
 	if cfg.Count < 1 || cfg.Count > 128 {
 		return nil, fmt.Errorf("invalid config param count: %v, want [1, 128]", cfg.Count)
 	}
+	hostTotalMemory := osutil.SystemMemorySize()
+	minMemory := uint64(cfg.Count) * 10_000_000
```

Comment:
Is it 10MB without 485760 bytes?;)

Reply:
Just as arbitrary :) I do actually want to know what a good default should be... I think around 50MB or 100MB might actually be a sensible minimum.